docs/_Docs_DomainsSkills.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Domains and Skills
+section: Docs
+order: 5
+---
+
+Domains and Skills are concepts that define a colony's organizational structure in a decentralized context, and allow for the division of labor without a strict management hierarchy.
+
+Domains are a structure for compartmentalizing the work and shared resources of a colony into smaller, more specialized sub-groups which are analogous to the departments of a traditional company.
+
+Skills are a similar structure that categorize the _type_ of work done, independent of the domain or colony in which the work took place.
+
+==TOC==
```

Comment:
This "==TOC==" markdown isn't presenting right. Not sure what it stands for

Reply:
This will automatically be replaced by the table of contents before rendering to html. Think of it as a placeholder